### DISCUSSION TITLE
Why @ is forbidden character in the password?

### DISCUSSION THREAD

**Discussion by VelizarVESSELINOV** [bluenviron/mediamtx#1580] [Questions] [answered]
Is "@" an RTSP protocol a forbidden character or just rtsp-simple-server definition of forbidden characters?

`contains unsupported characters (supported are A-Z,0-9,!,$,(,),*,+,.,;,<,=,>,[,],^,_,-,{,})`

**Comment by aler9** [comment, marked as answer]
Username and passwords now accept additional characters, including the AT character. PR: https://github.com/aler9/mediamtx/pull/1666

**Comment by aler9** [comment]
added in v0.22.1

[a comment by an automated account is not shown]
